fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -236,7 +239,7 @@ def build_upload_command(username, password, source = "/tmp", provider_short_nam
         "-u #{username.shellescape}",
         "-p #{password.shellescape}",
         "-f #{source.shellescape}",
-        ENV["DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS"], # that's here, because the user might overwrite the -t option
+        additional_upload_parameters, # that's here, because the user might overwrite the -t option
         '-t Signiant',
```

Comment:
Same here. This will give two `-t` parameters. Is that going to mess things up? 🤔